db/schema.rb

```diff
@@ -171,7 +171,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.uuid "school_id", null: false
     t.uuid "lead_provider_id", null: false
-    t.uuid "cohort_id", default: "f85b3d42-9f6e-41e2-a014-0a09caea7e93", null: false
+    t.uuid "cohort_id", default: "8fa3db39-3845-40ce-a4da-662223472891", null: false
```

Comment:
Where did this come from? It's also rather unusual to hardcode an id in the schema.

Reply:
Hmm, this will change every time someone deletes their local db which can lead to all sorts of mayhem. I think we need to remove the default and just explicitly require a cohort

Reply:
In the migration the default was set as `Cohort.first`

Reply:
I think I'll open a new PR to remove the default from this column?

Reply:
I was suggesting the same thing in the ecf channel

Reply:
I suggested it first 😉

Reply:
Point taken ;)